users/views.py

```diff
@@ -10,3 +12,22 @@ class SignUpView(CreateView):
     form_class = SignUpForm
     template_name = 'users/signup.html'
     success_url = '/'
+
+
+class FavedSeriesView(LoginRequiredMixin, ListView):
+    template_name = "users/followed_series.html"
+    context_object_name = 'faved_series_list'
+
+    login_url = '/accounts/login/'
+    redirect_field_name = ''
+
+    def get_queryset(self):
+        return APIShow.objects.filter(followers=self.request.user)
+
+    def get_redirect_url(self):
```

Comment:
This looks like a forgotten overridden method; is it actually necessary? (There's no form, it's just a list, so when are we redirecting?)